Makefile

```diff
@@ -14,6 +14,11 @@ GO_FILES=$(shell find . -type f -name '*.go' -not -path './vendor/*')
 setup_dev:
 	go get -u github.com/go-swagger/go-swagger/cmd/swagger
 	go get -u github.com/golang/dep/cmd/dep	
+	go get -u github.com/sirupsen/logrus
+	go get -u github.com/pkg/errors
+	go get -u github.com/lib/pq
+	go get -u github.com/jmoiron/sqlx/types
+	go get -u github.com/docker/go-units
```

Comment:
Actually we don't need to add dependencies here as they will be dragged in by `dep` during a build. 

see https://github.com/iateadonut/ledger/blob/master/Makefile#L52